### PR TITLE
Advanced Scattering Examples

### PR DESCRIPTION
**Pull request by dennis-lynch-nv**

### CODE REVIEW

File: omni.replicator/snippets/replicator_scatter_multi_trigger.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+def randomize_spheres():
+    # create small spheres to sample inside the plane
+    spheres = rep.create.sphere(scale=0.4, count=60)
+
+    # scatter small spheres
+    with spheres:
+        rep.randomizer.scatter_2d(
+            surface_prims=[plane_samp, sphere_samp],
+            no_coll_prims=[cylinder],
```
Comment (JenNVIDIA) [inline omni.replicator/snippets/replicator_scatter_multi_trigger.py:54]: When using Code 2022.3.3, error occurs: TypeError: scatter_2d() got an unexpected keyword argument 'no_coll_prims'

File: omni.replicator/snippets/scattering/scatter2D_multi_surface.py
Comment (JenNVIDIA) [inline omni.replicator/snippets/scattering/scatter2D_multi_surface.py:1]: Error in Code 2022.3.3

2023-07-21 23:29:35  [Error] [omni.graph.core.plugin] /Replicator/SDGPipeline/OgnScatter2D: OmniGraph Error: Scatter2D Error: Expected prim at /Replicator/SDGPipeline/OgnGroup_02/outputs_prims to be of type UsdGeom.Mesh or UsdGeom.GeomSubset but got type Output
2023-07-21 23:29:35  [Error] [omni.graph.core.plugin]                  (from compute() at line 124 in c:\ov\pkg\prod-code-2022.3.3\extscache\omni.replicator.core-1.7.7+104.2.wx64.r.cp37\omni\replicator\core\ogn\python\_impl\nodes\OgnScatter2D.py)

File: omni.replicator/snippets/scattering/scatter2D_with_limits.py
Comment (JenNVIDIA) [inline omni.replicator/snippets/scattering/scatter2D_with_limits.py:1]: 2023-07-21 23:30:45  [Error] [omni.kit.app.plugin] [py stderr]: TypeError: scatter_2d() got an unexpected keyword argument 'min_samp'

File: omni.replicator/snippets/replicator_scatter_multi_trigger.py
Comment (JenNVIDIA) [inline omni.replicator/snippets/replicator_scatter_multi_trigger.py:1]: For Code 2023.1.0-rc.12 
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin] [py stderr]: NotImplementedError: Parameterizing of node Node("/Replicator/SDGPipeline/OgnSetVisibility") used in multiple triggers ({Node("/Replicator/SDGPipeline/OgnOnFrame_01"), Node("/Replicator/SDGPipeline/OgnOnFrame")}) not currently supported.
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin] 
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin] At:
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin]   c:/ov/data/kit/code/2023.1/exts/3/omni.replicator.core-1.9.8+105.0.wx64.r.cp310/omni/replicator/core/scripts/utils/utils.py(548): _attach_scheduled_exec
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin]   c:/ov/data/kit/code/2023.1/exts/3/omni.replicator.core-1.9.8+105.0.wx64.r.cp310/omni/replicator/core/scripts/utils/utils.py(646): __init__
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin]   c:/ov/data/kit/code/2023.1/exts/3/omni.replicator.core-1.9.8+105.0.wx64.r.cp310/omni/replicator/core/scripts/utils/utils.py(606): __call__
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin]   c:/users/jborucki/appdata/local/temp/xn9w.0/script_1689982547.py(41): <module>
2023-07-21 23:35:49  [Error] [omni.kit.app.plugin]

File: omni.replicator/snippets/scattering/scatter2D_multi_surface.py
Comment (JenNVIDIA) [reply omni.replicator/snippets/scattering/scatter2D_multi_surface.py:1]: Works in Code 2023.1.1-rc.12

File: omni.replicator/snippets/scattering/scatter2D_with_limits.py
Comment (JenNVIDIA) [reply omni.replicator/snippets/scattering/scatter2D_with_limits.py:1]: Works in Code 2023.1.1-rc.12

File: omni.replicator/snippets/replicator_scatter_multi_trigger.py (hunk 1 above)
Comment (dennis-lynch-nv) [reply omni.replicator/snippets/replicator_scatter_multi_trigger.py:54]: Yes, these examples are for Replicator 1.9.9+, still pending release

**Review by JenNVIDIA — APPROVED** [review]

**Pull request merged**